sdk/template/template/review/template.api.md

```diff
@@ -19,6 +19,7 @@ export class ConfigurationClient {
 export interface ConfigurationClientOptions extends PipelineOptions {
 }
 
+
```

Comment:
This file shouldn't be part of this PR since we aren't touching the template package.